Kickstarter-iOS/Views/Storyboards/BetaTools.storyboard

```diff
@@ -64,17 +62,17 @@
                                         <rect key="frame" x="0.0" y="108.5" width="343" height="29"/>
                                         <subviews>
                                             <label opaque="NO" userInteractionEnabled="NO" contentMode="left" text="Change environment" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Hcy-KX-D9W">
-                                                <rect key="frame" x="0.0" y="0.0" width="259" height="29"/>
+                                                <rect key="frame" x="0.0" y="0.0" width="252" height="29"/>
                                                 <fontDescription key="fontDescription" name=".AppleSystemUIFont" family=".AppleSystemUIFont" pointSize="17"/>
-                                                <color key="textColor" red="0.39607843139999999" green="0.41176470590000003" blue="0.41176470590000003" alpha="1" colorSpace="calibratedRGB"/>
+                                                <color key="textColor" red="0.0" green="0.0" blue="0.0" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
Updated the Storyboard in order to better reflect final appearance. The previous one wasn't very good as it didn't even use consistent colors for all buttons / labels.